### PR TITLE
Evgeluev/jel nginx load balancer redirect

### PR DESCRIPTION
**Pull request by evgeluev**

### CODE REVIEW

File: content/en/docs/jelastic-paas/guides/nginx_restrict_access.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Nginx access restriction"
```
Comment (Doctor-love) [inline content/en/docs/jelastic-paas/guides/nginx_restrict_access.md:2]: There already exist a PR for this guide: #56

File: content/en/docs/jelastic-paas/guides/nginx_load_balancer_redirect.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Nginx Load Balancer Redirect"
```
Comment (Doctor-love) [inline content/en/docs/jelastic-paas/guides/nginx_load_balancer_redirect.md:2]: Maybe something like "Nginx HTTP to HTTPS redirect

**Review by Doctor-love — CHANGES_REQUESTED** [review]

**Review by MrMoreau — COMMENTED** [review]
+1 on changing the title, otherwise it LGTM

**Review by Doctor-love — APPROVED** [review]
Let's get this merged!

**Pull request merged**